modules/c2distro_core/c2distro_core.module

```diff
@@ -87,3 +87,15 @@ function _c2distro_core_set_thumbnail_alt($entity) {
 function c2distro_core_media_source_info_alter(array &$sources) {
   $sources['file']['class'] = 'Drupal\c2distro_core\Plugin\media\Source\File';
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_update().
+ * 
+ * Invalidate cache tags after hero block content type has modified.
+ */
+function c2distro_core_block_content_update(Drupal\Core\Entity\EntityInterface $entity) {
```

Comment:
Add:
use Drupal\Core\Entity\EntityInterface

and 
function c2distro_core_block_content_update(EntityInterface $entity) {